src/app/components/apps-view/apps-view.component.html

```diff
@@ -181,18 +164,12 @@ <h3>Allocations</h3>
 
         <mat-row *matRowDef="let row; columns: allocColumnIds"></mat-row>
 
-        <mat-footer-row
-          *matFooterRowDef="['noRecord']"
-          [ngStyle]="{ display: isAllocDataSourceEmpty() ? '' : 'none' }"
-        ></mat-footer-row>
+        <mat-footer-row *matFooterRowDef="['noRecord']" [ngStyle]="{ display: isAllocDataSourceEmpty() ? '' : 'none' }">
+        </mat-footer-row>
       </mat-table>
 
-      <mat-paginator
-        #allocationMatPaginator
-        [pageSizeOptions]="[10, 20, 50, 100]"
-        [ngStyle]="{ display: isAllocDataSourceEmpty() ? 'none' : '' }"
-        showFirstLastButtons
-      ></mat-paginator>
+      <mat-paginator #allocationMatPaginator [pageSizeOptions]="[10, 20, 50, 100]"
+        [ngStyle]="{ display: isAllocDataSourceEmpty() ? 'none' : '' }" showFirstLastButtons></mat-paginator>
     </div>
   </div>
-</div>
+</div>
```

Comment:
please make sure we have a newline at the end of the file.

Reply:
Sure, added